addon/mixins/recognizers.js

```diff
@@ -29,6 +37,8 @@ export default Mixin.create({
   },
 
   __setupRecognizers: on('didInsertElement', function() {
+    if (this.get('__fastboot.isFastBoot')) { return; }
```

Comment:
@eriktrom I was helping @anthonybalmeo on this. You're right, `didInsertElement` should not be called in FastBoot, but this code was somehow getting run, hence the guard. Maybe it is indirectly called by `init` like you suggest below. We're looking into that and will let you know.